src/components/FloatingTip/FloatingTip.tsx

```diff
@@ -24,6 +24,7 @@ export type FloatingTipProps = {
     event: React.MouseEvent<HTMLDivElement, MouseEvent>,
     reason: ModalCloseReason | FloatingTipCloseReason,
   ) => void;
+  popOverProps?: Partial<PopoverProps>;
```

Comment:
- （この props に限らず今回加えた各差分の props に）Doc コメントがあると親切
- `popOver` じゃなくて `popover` の方が良さそう
  - 既存がそうなので合わせた方が無難
  - 少なくとも UI の文脈では pop-over って一般的には呼ばない
 
```suggestion
  /**
   * props of [Popover](/?path=/docs/components-utils-popover--example)
   */
  popoverProps?: Partial<PopoverProps>;
```